### ISSUE TITLE
Someone is claiming your work as their own.

### ISSUE THREAD

**Issue by henzosabiq** [nickgerace/gfold#264]
Hi. I thought you might want to know about this. 

This person <https://github.com/trinhminhtriet> copied your entire repository almost entirely verbatim, except he changed references (project names, links, etc) of the original to theirs and removed attributions.

Here is the repo: <https://github.com/trinhminhtriet/gitmgr>

They made a thread on r/rust before to promote his other stolen project (<https://www.reddit.com/r/rust/comments/1gyngfl/introducing_rmrfrs_a_powerful_filesystem_cleaning/>). The project was under his profile (<https://github.com/trinhminhtriet/rmrfrs>). When some users pointed out the plagiarism, he did not bother addressing anything, deleted the thread, closed issues made by other users in his stolen repo, and made a proxy org 'rust-rs' to try to obscure the footstep. One user pointed out that his other repo (yours) was also stolen.

Archived links in case they delete their account:
<http://web.archive.org/web/20241124164634/https://github.com/trinhminhtriet>
<https://web.archive.org/web/20241124165127/https://old.reddit.com/r/rust/comments/1gyngfl/introducing_rmrfrs_a_powerful_filesystem_cleaning/>
<https://web.archive.org/web/20241124172705/https://github.com/trinhminhtriet/gitmgr>

Edit: fixed wrong project links

**Comment by henzosabiq** [comment]
> Can we report this to GitHub?

I tried to report it under Copyright Infringement earlier, but only the repo owner can do that. They need to submit a DMCA takedown notice themselves.

**Comment by nickgerace** [comment]
Thank you both very much. @henzosabiq this was very kind of you and great to do. I'll look into how to report it.

**Comment by henzosabiq** [comment]
> Thank you both very much. @henzosabiq this was very kind of you and great to do. All the evidence must have taken awhile to compile and I sincerely appreciate it. I'll look into how to report it.

No problem, I'm glad this could help. I hope this gets resolved quickly.

**Comment by nickgerace** [comment]
@skewb1k I agree. Yes please let me/us know through this issue if you do. Thank you.
